_layouts/event-page.html

```diff
@@ -0,0 +1,137 @@
+---
+layout: default
+---
+
+<article class="event-page">
+  <header class="event-header">
+    {% if page.featured_image %}
+    <div class="featured-image">
+      <img src="{{ page.featured_image }}" alt="{{ page.title }}">
```

Comment:
RE: image handling

There's a helper include for this already:

```
{% include image.html
file="photo.jpg"
alt="alt text"
%}
```

That's for main page in-line images. So that may or may not work for this use case.

Otherwise, for handling image `src`, we need to use this:

```
{% link {{ page.featured_image | img_url }} %}
```

Explanation:

- `{% link %}` validates that the file exists. If it's missing, `jekyll build` will fail. This will catch errors.
- `| img_url` forms the correct path to the file without having to hard-code paths

Basically, Jekyll has a bunch of built-in functionality to prevent missing/broken links so we should always avoid using raw paths.

Reply:
For `img/` subdirectories:

Suppose we have `img/retreat2025/`

`{% link {{ page.featured_image | img_url: "retreat2025" }} %}`